shell/browser/electron_browser_context.h

```diff
@@ -81,6 +81,9 @@ class ElectronBrowserContext : public content::BrowserContext {
       bool in_memory,
       base::DictionaryValue options = base::DictionaryValue());
 
+  static ElectronBrowserContext* FromBrowserContext(
```

Comment:
We don't usually add helpers when it is just a simple `static_cast`.

Reply:
I've gone with the `static_cast` and removed the helper.